calibre.py

```diff
@@ -0,0 +1,164 @@
+from sqlalchemy import create_engine
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import sessionmaker
+from datetime import datetime
+
+import click
+
+# TBD datum přidání do knihovny
+# TBD přečtená
+# TBD rozečtená
+# TBD datum zapůjčení
+
+'''
+Aplikace by měla uchovávat tyto údaje:
+- název knihy
+- autor
+- počet stran
+- datum přidání do knihovny
+- přečtená (ano/ne)
+- rozečtená (ano/ne)
+- půjčená (půjčená komu/None)
+- datum zapůjčení (datum/None)
+- další informace, které uznáte za vhodné (např. identifikace knihy, slovní hodnocení)
+
+S aplikací by mělo být možné zacházet zeširoka,
+tedy knihy vkládat, vypisovat, zapůjčovat a zápůjčky vracet,
+editovat chybné položky, nastavovat příznak přečtení a rozečtení atd.
+Výpis ideálně nějak rozumně formátovaný,
+řípadně např. celkový + detailní o jedné knize.
+'''
+
+db = create_engine("sqlite:///knihy.sqlite")
+Base = declarative_base()
+
+def pripoj_se():
+    Base.metadata.create_all(db)
+    Session = sessionmaker(bind=db)
+    return Session()
+
+@click.group()
+def knihovna():
+    pass
+
+
+class Kniha(Base):
+    # Název tabulky s knihami
+    __tablename__ = "knihy"
+    # Číselný identifikátor knihy
+    id = Column(Integer, primary_key=True)
+    # Název knihy
+    nazev = Column(String)
+    # Autor
+    autor = Column(String)
+    # Počet stran
+    pocet_stran = Column(Integer)
+    # Datum přidání do knihovny
+    pridano = Column(DateTime)
+    # Přečtená (ano/ne)
+    precteno = Column(String)
```

Comment:
pokud v IT svete neco nabyva dvou hodnot, preferuje se prevest to na 0 a 1, nebo True a False, dal v kodu s tim pak muzes pracovat jako:
`if True: 
text='precteno' 
else:
text='neprecteno`